browser/ui/views/tabs/brave_browser_tab_strip_controller.cc

```diff
@@ -45,6 +53,24 @@ void BraveBrowserTabStripController::SetCustomTitleForTab(
       ->SetCustomTitleForTab(index, title);
 }
 
+void BraveBrowserTabStripController::OpenEmojiPickerForTab(int index) {
```

Comment:
the tab strip controller is responsible for managing the tab strip, not for opening emoji pickers. Please do not add functionality to classes that is clearly outside of it's responsibilities.